### ISSUE TITLE
gnome-documents fails to start

### ISSUE THREAD

**Issue by maxice8** [voidlinux/void-packages#8242]
```
$ gnome-documents
Unsatisfied dependency: GdPrivate
```

How to reproduce:

1. Install gnome-documents ( xbps-install -S gnome-documents )
2. Try to run it

**Comment by Gottox** [comment]
maybe related to #8182.

**Comment by maxice8** [comment]
Thanks for the pointer, i'll try again when #8121 effects hit the repo

**Comment by maxice8** [comment]
No, the addition of tracker-miners doesn't seem to fix the problem

**Comment by greywolf01** [comment]
Over here no problems opening up Documents.

**Comment by maxice8** [comment]
On Tue, Oct 17, 2017 at 08:19:19PM +0000, greywolf01 wrote:
> Over here no problems opening up Documents.
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/voidlinux/void-packages/issues/8242#issuecomment-337357713

Alright, i'll assume it's due to me not running a full gnome-environment ( gnome-shell )

**Comment by Gottox** [comment]
Actually, this is still an issue. gnome-documents should have all dependencies that are needed to run it.
